.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
```

Comment:
In general I think changes like this should get their own PR for visibility.  No biggie but is food for thought on future ones.  We had a bunch of discussions about this several months ago before you started and how it makes it hard to revert if issues creep in inadvertently.

Reply:
I agree, will try to revert. I remember some related issues, but not 100% sure